multitenancy/src/main/java/io/micronaut/multitenancy/filter/TenantResolverFilterConfigurationProperties.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2017-2024 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.multitenancy.filter;
+
+import io.micronaut.context.annotation.ConfigurationProperties;
+import io.micronaut.core.annotation.Internal;
+import io.micronaut.multitenancy.MultitenancyConfiguration;
+
+/**
+ * {@link ConfigurationProperties} implementation of {@link TenantResolverFilterConfiguration}.
+ */
+@ConfigurationProperties(TenantResolverFilterConfigurationProperties.PREFIX)
+@Internal
+final class TenantResolverFilterConfigurationProperties implements TenantResolverFilterConfiguration {
+    public static final String PREFIX = MultitenancyConfiguration.PREFIX + ".filter";
+    public static final String PROPERTY_ENABLED = PREFIX + ".enabled";
+    /**
+     * The default regex pattern.
+     */
+    @SuppressWarnings("WeakerAccess")
+    public static final String  DEFAULT_REGEX_PATTERN = "^.*$";
```

Comment:
```suggestion
    public static final String DEFAULT_REGEX_PATTERN = "^.*$";
```

Is checkstyle on? I think it would've caught this?